docs/source/WritingTests.rst

```diff
@@ -992,6 +992,82 @@ Now, trying to list the tests on the ``mytest.py`` file again::
 You can also use the ``:avocado: disable`` tag, that works the opposite way:
 Something looks like an Avocado test, but we force it to not be listed as one.
 
+Unittest Compatibility Limitations And Caveats
+==============================================
+
+The Avocado test runner uses a different approach to running tests
```

Comment:
OK, how about:

"When executing tests, Avocado uses different techniques than most
other Python unittest runners.  This brings some compatibility
limitations that Avocado users should be aware."